.github/workflows/main.yml

```diff
@@ -15,71 +15,13 @@ on:
     - cron:  '42 7 * * *'
 
 jobs:
-  test-benchmark:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        include:
-          - os: ubuntu-latest
-          - os: macos-latest
-    env:
-      CONDA_ENV: 'test_env'
-      VERSION_PYTHON: ${{ matrix.version_python }}
-      BENCHOPT_BRANCH: benchopt:main
-      BENCHOPT_DEBUG: 1
-
-    defaults:
-      run:
-        # Need to use this shell to get conda working properly.
-        # See https://github.com/marketplace/actions/setup-miniconda#important
-        shell: bash -l {0}
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Setup Conda
-      uses: conda-incubator/setup-miniconda@v2
-      with:
-        activate-environment: ${{ env.CONDA_ENV }}
-        python-version: 3.8
-        # Use miniforge to only get conda-forge as default channel.
-        miniforge-version: latest
-
-    - run: conda info
-
-     # see https://github.com/benchopt/benchmark_logreg_l1/issues/20
-    - name: Install OpenMP
-      if: matrix.os == 'macos-latest'
-      run: |
-        brew install libomp
-
-    - name: Install benchopt and its dependencies
-      run: |
-        conda info
-        conda install -yq pip
-
-        # Get the correct branch of benchopt
-        user=${BENCHOPT_BRANCH%:*}
-        branch=${BENCHOPT_BRANCH##*:}
-        pip install -U git+https://github.com/$user/benchopt@$branch
-
-    - name: Test
-      run: |
-        export OMP_NUM_THREADS=1
-        benchopt test . --env-name bench_test_env -vl
-        benchopt test . --env-name bench_test_env -vl --skip-install
-
-
-  linter-flake8:
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-
-    - name: Lint with flake8
-      run: |
-        pip install flake8
-        flake8 .
+  benchopt_dev:
+    uses: benchopt/template_benchmark/.github/workflows/test_benchmarks.yml@main
+    with:
+      benchopt_branch: benchopt@main
+  benchopt_release:
+    uses: benchopt/template_benchmark/.github/workflows/test_benchmarks.yml@main
+    with:
+      benchopt_version: latest
+  lint:
+    uses: benchopt/template_benchmark/.github/workflows/lint_benchmarks.yml@main
```

Comment:
@Badr-MOUFAD cosmetic remark: add a new line at the end of file

Reply:
I'm unable to add a new line. It seems that it's removed automatically

Also I checked other workflows, they are all missing a new line
- https://github.com/scikit-learn-contrib/skglm/blob/main/.github/workflows/main.yml
- https://github.com/benchopt/benchmark_logreg_l2/blob/main/.github/workflows/main.yml
- https://github.com/mathurinm/celer/blob/main/.github/workflows/flake8.yml